canonical-web-and-design/global-nav
# Improve accessibility of global nav

## Done

- Add 'Skip to main content' link
- Added title element to logo SVG
- Improved accessibility of example page to eliminate false positives in testing

## QA

- Check example page and run [aXe](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd) dev tools test

<img width="639" alt="Screenshot 2019-05-09 at 16 27 24" src="https://user-images.githubusercontent.com/505570/57465854-60125680-7277-11e9-82d2-f1776f00b123.png">
